src/DetailsView/reports/components/report-sections/rule-detail.tsx

```diff
@@ -17,16 +25,64 @@ export type RuleDetailProps = {
 };
 
 export const RuleDetail = NamedSFC<RuleDetailProps>('RuleDetails', props => {
```

Comment:
Seems like all this component renders is the group header, do we want to consider renaming this component?

Reply:
I'm not sure... but, this is just part of the changes I'm making for fix the collapsible control so... I can rename later. Not to worried about it.